modules/24_trade/se_trade/datainput.gms

```diff
@@ -212,33 +230,137 @@ $ifthen.import_nzero_EU "%cm_import_EU%" == "nzero"
+
+*** exponential curve for years in between
+  p24_seTradeCapacity(t,regi,regi2,"seh2")$(p24_seTradeCapacity("2050",regi,regi2,"seh2") and (t.val gt 2030 and t.val lt 2050)) = 
+    p24_seTradeCapacity("2030",regi,regi2,"seh2") * ((sqrt(sqrt(p24_seTradeCapacity("2050",regi,regi2,"seh2") / p24_seTradeCapacity("2030",regi,regi2,"seh2")))) ** ((t.val - 2030)/5)) ;
+
+*** E-fuels (e-liquids) trade:
+***   All regions (EU-27 and UKI) import proportionally to their 2050 FE|Transport|Pass|Aviation + FE|Industry|Chemicals|Liquids in the reference NPi run.
+***   Exporting regions: SSA, LAM and MEA (one-third each)
+***   Import quantities: exponential increase from 0.6 TWh/yr by 2030 to 50 TWh/yr by 2050 for Germany
 
 *** defining Germany seliqsyn trade import flows
-  loop(regi$(sameas(regi,"SSA") or sameas(regi,"LAM") or sameas(regi,"MEA")), !! each supplier region provide one-third of total imports
+  loop(regi$(sameas(regi,"SSA") or sameas(regi,"LAM") or sameas(regi,"MEA")),
 *** 2050 and onward
     p24_seTradeCapacity(t,regi,regi2,"seliqsyn")$(t.val ge 2050) = 
-      ( (100 / sm_TWa_2_TWh)  / p24_aviationAndChemicalsFEShareInRegion("2050","EUR_regi","DEU") ) !! total EUR imports based on Germany values
-      * p24_aviationAndChemicalsFEShareInRegion("2050","EUR_regi",regi2) * 1/3;
+      ( (50 / sm_TWa_2_TWh)  / p24_aviationAndChemicalsFEShareInRegion("2050","EUR_regi","DEU") ) !! total EUR imports based on Germany values
+      * p24_aviationAndChemicalsFEShareInRegion("2050","EUR_regi",regi2) 
+      * 1/3; !! each supplier region provide one-third of total imports
 *** 2030 
-    p24_seTradeCapacity("2030",regi,regi2,"seliqsyn") = (((1.2 / sm_TWa_2_TWh) / p24_aviationAndChemicalsFEShareInRegion("2050","EUR_regi","DEU") ) * p24_aviationAndChemicalsFEShareInRegion("2050","EUR_regi",regi2)) * 1/3;
+    p24_seTradeCapacity("2030",regi,regi2,"seliqsyn") = 
+      ( (0.6 / sm_TWa_2_TWh) / p24_aviationAndChemicalsFEShareInRegion("2050","EUR_regi","DEU") )
+      * p24_aviationAndChemicalsFEShareInRegion("2050","EUR_regi",regi2) 
+      * 1/3; 
   );
 
 *** exponential curve for years in between
   p24_seTradeCapacity(t,regi,regi2,"seliqsyn")$(p24_seTradeCapacity("2050",regi,regi2,"seliqsyn") and (t.val gt 2030 and t.val lt 2050)) = 
     p24_seTradeCapacity("2030",regi,regi2,"seliqsyn") * ((sqrt(sqrt(p24_seTradeCapacity("2050",regi,regi2,"seliqsyn") / p24_seTradeCapacity("2030",regi,regi2,"seliqsyn")))) ** ((t.val - 2030)/5)) ;
 
+
+*** bio-liquids trade:
+***   All regions (EU-27 and UKI) import proportionally to their 2050 FE|Transport|Pass|Aviation + FE|Industry|Chemicals|Liquids in the reference NPi run.
+***   Exporting regions: SSA and LAM (half each)
+***   Import quantities: exponential increase from 0.3 EJ by 2030 to 7.44 EJ by 2050 for EU-27
+
+*** defining EU-27 & UK seliqbio trade import flows
+  loop(regi$(sameas(regi,"SSA") or sameas(regi,"LAM")),
```

Comment:
will bioliquids actually come (only) from LAM and SSA? 
I would expect OAS (palmoil from Indonesia) to have a much larger share compared to SSA, so I would at least split it by three and include OAS?

on second thought I would simply look to the PE biomass trade flows and see who is exporting in 2050 in 650 scenarios.

Reply:
I used this paper as reference to set LAM and SSA as the exporting regions:
https://onlinelibrary.wiley.com/doi/10.1111/gcbb.12614

![image](https://github.com/remindmodel/remind/assets/33692508/39314aa9-c884-4aef-b0ac-555f6dba29ce)
Figure 7. Regional bioenergy potential in 2050 under each scenario

Reply:
I think that due to geographical location, south east asia would most probably be the preferable source for China and Japan potential imports instead of Europe.

Reply:
right. 
could you write "LAM and SSA as exporting regions were chosen based on this paper: doixxxx" so that when this is revised in the future, we know how the decision was taken?

then I am happy :-)